Nerfs executioner sword, lightly buffs the shortsword

BobJoga opened the pull request:
## About The Pull Request

Executioner sword will now only delimb extremities - it won't disembowel or decap, and it will always respect armor - only limbs that have less than 25 melee armoring can be dismembered.

Shortsword now has 0.65x the click delay, instead of 0.75x.

## Why It's Good For The Game

balance

## Changelog
:cl:
balance: Executioner sword now respects armor and is unable to decap or disembowel.
balance: The shortsword is slightly faster at attacking.
/:cl:

BobJoga commented:
> ![image](https://user-images.githubusercontent.com/53862927/85472574-ec7df680-b5a9-11ea-9794-dd4a1edf0616.png)
> This is the first click I did on a human mob when targeting chest.
> Second mob took six clicks to spill internal organs, again targeting the chest.
> Targeting the right arm it took 6 hits to dismember. No combat mode.
> Targeting the head, I dropped them with 20 hits with no combat mode, no decaps, disembowling or delimbs, they just went into softcrit.
> Fifth test, same result as last. Hitting them in the head did nothing.
> Sixth test. Spilled internals in two clicks. Targeting chest.
> Seventh test. Spilled internals in two clicks. Chest.
> Eight test. Spilled internals in four clicks. Chest again.
> Ninth, three clicks, chest.
> 
> Alright so I think I don't have to keep clicking on people. The same problem still exists, except instead of getting an instakill where you also get the head to conveniently pick up and destroy, they just lose their organs and die a few seconds later.
> 
> Regardless of instakill being fixed, there's a way to do this that is more fun for everyone on either side of the screen.
> Method 1:
> Make a check for limb damage and only have it roll random chance to delimb on mangled limbs. Up the force to compensate it.
> Method 2:
> Up the force to the same as the spear with a much lighter click delay (1.1* to 1.2*) once wielded. Have it roll random chance to delimb only once wielded and limbs are mangled. This'll stop you just using a welder, clicking 4 times, then using the sword to bet on delimbs. Keep in mind at the same damage as the spear, being able to put this in your belt slot is very useful. Having to dedicate your backpack or hand to carrying spears is major downside this won't have.
> Method 3:
> Remove delimbs entirely and just make this 20 force or something, but I assume your entire purpose of this weapon was to have the delimb gimmick.
> 
> You're of course free to just go to Azarak, Useroth, Afya, Bunny or Jake if you don't want to do anything other than move the one-click disembowelling.

By default sharp weapons can delimb if the limb is sufficiently damaged. These balancing options just take away from the gimmick of the sword.

Yawet330 commented:
> > ![image](https://user-images.githubusercontent.com/53862927/85472574-ec7df680-b5a9-11ea-9794-dd4a1edf0616.png)
> > This is the first click I did on a human mob when targeting chest.
> > Second mob took six clicks to spill internal organs, again targeting the chest.
> > Targeting the right arm it took 6 hits to dismember. No combat mode.
> > Targeting the head, I dropped them with 20 hits with no combat mode, no decaps, disembowling or delimbs, they just went into softcrit.
> > Fifth test, same result as last. Hitting them in the head did nothing.
> > Sixth test. Spilled internals in two clicks. Targeting chest.
> > Seventh test. Spilled internals in two clicks. Chest.
> > Eight test. Spilled internals in four clicks. Chest again.
> > Ninth, three clicks, chest.
> > Alright so I think I don't have to keep clicking on people. The same problem still exists, except instead of getting an instakill where you also get the head to conveniently pick up and destroy, they just lose their organs and die a few seconds later.
> > Regardless of instakill being fixed, there's a way to do this that is more fun for everyone on either side of the screen.
> > Method 1:
> > Make a check for limb damage and only have it roll random chance to delimb on mangled limbs. Up the force to compensate it.
> > Method 2:
> > Up the force to the same as the spear with a much lighter click delay (1.1* to 1.2*) once wielded. Have it roll random chance to delimb only once wielded and limbs are mangled. This'll stop you just using a welder, clicking 4 times, then using the sword to bet on delimbs. Keep in mind at the same damage as the spear, being able to put this in your belt slot is very useful. Having to dedicate your backpack or hand to carrying spears is major downside this won't have.
> > Method 3:
> > Remove delimbs entirely and just make this 20 force or something, but I assume your entire purpose of this weapon was to have the delimb gimmick.
> > You're of course free to just go to Azarak, Useroth, Afya, Bunny or Jake if you don't want to do anything other than move the one-click disembowelling.
> 
> By default sharp weapons can delimb if the limb is sufficiently damaged. These balancing options just take away from the gimmick of the sword.

![image](https://user-images.githubusercontent.com/65188584/85502708-cb37fd00-b5df-11ea-8b5d-e9cfbe7c5864.png)

sharpened exe sword literally 1 hit beheads though...

BobJoga commented:
> > > ![image](https://user-images.githubusercontent.com/53862927/85472574-ec7df680-b5a9-11ea-9794-dd4a1edf0616.png)
> > > This is the first click I did on a human mob when targeting chest.
> > > Second mob took six clicks to spill internal organs, again targeting the chest.
> > > Targeting the right arm it took 6 hits to dismember. No combat mode.
> > > Targeting the head, I dropped them with 20 hits with no combat mode, no decaps, disembowling or delimbs, they just went into softcrit.
> > > Fifth test, same result as last. Hitting them in the head did nothing.
> > > Sixth test. Spilled internals in two clicks. Targeting chest.
> > > Seventh test. Spilled internals in two clicks. Chest.
> > > Eight test. Spilled internals in four clicks. Chest again.
> > > Ninth, three clicks, chest.
> > > Alright so I think I don't have to keep clicking on people. The same problem still exists, except instead of getting an instakill where you also get the head to conveniently pick up and destroy, they just lose their organs and die a few seconds later.
> > > Regardless of instakill being fixed, there's a way to do this that is more fun for everyone on either side of the screen.
> > > Method 1:
> > > Make a check for limb damage and only have it roll random chance to delimb on mangled limbs. Up the force to compensate it.
> > > Method 2:
> > > Up the force to the same as the spear with a much lighter click delay (1.1* to 1.2*) once wielded. Have it roll random chance to delimb only once wielded and limbs are mangled. This'll stop you just using a welder, clicking 4 times, then using the sword to bet on delimbs. Keep in mind at the same damage as the spear, being able to put this in your belt slot is very useful. Having to dedicate your backpack or hand to carrying spears is major downside this won't have.
> > > Method 3:
> > > Remove delimbs entirely and just make this 20 force or something, but I assume your entire purpose of this weapon was to have the delimb gimmick.
> > > You're of course free to just go to Azarak, Useroth, Afya, Bunny or Jake if you don't want to do anything other than move the one-click disembowelling.
> > 
> > 
> > By default sharp weapons can delimb if the limb is sufficiently damaged. These balancing options just take away from the gimmick of the sword.
> 
> ![image](https://user-images.githubusercontent.com/65188584/85502708-cb37fd00-b5df-11ea-8b5d-e9cfbe7c5864.png)
> 
> sharpened exe sword literally 1 hit beheads though...

I've fixed it. Sorry.
